### ISSUE TITLE
Animated thumb glow?

### ISSUE THREAD

**Issue by hacker1024** [suragch/audio_video_progress_bar#2]
Would it be possible to animate the thumb glow as it opens, similar to the material `Slider`?

**Comment by suragch** [comment]
That is certainly possible. I have some other things I'm working on right now but if you'd like to implement it I'd be glad to add it in.

**Comment by BraveEvidence** [comment]
This will help https://www.youtube.com/watch?v=IMQdSTlTXjA

**Comment by RenautMestdagh** [comment]
Any updates on this?

**Comment by suragch** [comment]
> Any updates on this?

No, is this something you're interested in working on? I'd start by reading the source code for the Flutter Slider widget and see how they implemented it.
